package.json

```diff
@@ -12,7 +12,8 @@
 		"export": "next export",
 		"test:unit": "jest --passWithNoTests",
 		"storybook": "start-storybook -p 6006",
-		"build-storybook": "build-storybook"
+		"build-storybook": "build-storybook",
+		"chromatic": "chromatic --exit-zero-on-changes"
```

Comment:
we need a project token to run this, will we just distribute this token or isn't needed to run chromatic locally?

Reply:
A token is needed to run it locally. But normally you shouldn't run it locally, as it's ran in the CI.
This script was added when I was following the setup guide from chromatic, but we can probably remove it unless the CI action uses it or something.